README.md

```diff
@@ -1,8 +1,16 @@
-# OpenSourceHelpCommunity.github.io
+# Open Source Help Community
+
+![OSHC](https://avatars0.githubusercontent.com/u/23719480?v=3&s=200)
+
+Our goal is to provide an environment where people who work on Open Source projects can gather together to discuss them([What is Open Source?](https://opensource.guide/how-to-contribute/)). 
```

Comment:
This looks good. Instead of linking this article link the repository under this organization. 

Feel free to submit links to [this](https://github.com/tapasweni-pathak/Getting-Started-With-Contributing-to-Open-Sources) repository if you find something which can help others.